src/components/popup/popup.tsx

```diff
@@ -49,40 +65,48 @@ export class Popup extends React.Component<PopupCompProps> {
         return null;
     }
 
+    public componentDidUpdate() {
+        const oldStyle = this.state.style;
```

Comment:
I don't think you need to put the old style in a variable if you only use it once.